### PR TITLE
Redmine #4568 Preserve MLPPP settings when saving interface settings RELENG_2_2

### PR DESCRIPTION
**Pull request by phil-davis**
Fix for RELENG_2_2 to match master pull request https://github.com/pfsense/pfsense/pull/1822

### CODE REVIEW

**Comment by phil-davis** [comment]
RELENG_2_2 branch seems to still have the old folder structure - good stuff, this PR can still merge there if needed.

File: usr/local/www/interfaces.php [hunk 1]
```diff
@@ -544,20 +544,20 @@ function remove_bad_chars($string) {
 			break;
 		case "pptp":
 			if ($_POST['pptp_dialondemand']) {
-				$reqdfields = explode(" ", "pptp_username pptp_password pptp_local pptp_subnet pptp_remote pptp_dialondemand pptp_idletimeout");
+				$reqdfields = explode(" ", "pptp_username pptp_password pptp_localip0 pptp_subnet0 pptp_remote0 pptp_dialondemand pptp_idletimeout");
```
Comment (rbgarga) [inline usr/local/www/interfaces.php]: pptp_localip0 should be pptp_local0 here, no?

File: usr/local/www/interfaces.php [hunk 2]
```diff
@@ -544,20 +544,20 @@ function remove_bad_chars($string) {
...
 				$reqdfieldsn = array(gettext("PPTP username"),gettext("PPTP password"),gettext("PPTP local IP address"),gettext("PPTP subnet"),gettext("PPTP remote IP address"),gettext("Dial on demand"),gettext("Idle timeout value"));
 			} else {
-				$reqdfields = explode(" ", "pptp_username pptp_password pptp_local pptp_subnet pptp_remote");
+				$reqdfields = explode(" ", "pptp_username pptp_password pptp_localip0 pptp_subnet0 pptp_remote0");
```
Comment (rbgarga) [inline usr/local/www/interfaces.php]: Same here, pptp_local0

**Comment by phil-davis** [comment]
Thanks for spotting. My brain has gone to mush with various versions of things for master and RELENG_2_2 and manually re-doing changes over time. This one has been sitting here for a while without the error being spotted.
I made pull request https://github.com/pfsense/pfsense/pull/1928 with a clean and correct (I hope) version of this for RELENG_2_2.

**Pull request closed without merging**